lib/web/mage/adminhtml/form.js

```diff
@@ -441,6 +452,17 @@ define([
                     if (!from || isInArray && isNegative || !isInArray && !isNegative) {
                         shouldShowUp = false;
                     }
+                // Check if radio button
+                } else {
+                    values = valuesFrom[idFrom].values;
+                    fromId = $(idFrom + values[0]);
+                    radioFrom = $$('[name="' + fromId.name + '"]:checked');
+                    isInArray = values.indexOf(radioFrom[0].value) != -1; //eslint-disable-line
```

Comment:
Add check on `radioFrom.length > 0`, make strict equality and remove `eslint-disable-line`